OOStepByStep/ClassRoom.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OOStepByStep
+{
+    public partial class ClassRoom
+    {
+        private int number;
+        private Teacher teacher;
+        private List<Student> students;
+
+        public ClassRoom(int number)
+        {
+            this.number = number;
+        }
+
+        public string IntroduceSelf(Person person)
```

Comment:
this behavior should move to teacher or student
e.g teacher.IntroduceSelf(), student.IntroduceSelf()